.github/styles/Vocab/Docs/accept.txt

```diff
@@ -11,6 +11,8 @@ Avro
 Conduktor
 [Cc]onnect
 Couchbase
+CPU
+cron
```

Comment:
is "cron" a word? I would suggest if we are referring to the command, it should be in code markup, so it won't need to be added to the spellchecker

Reply:
@lornajane in this case it is referring to CRON expression
https://en.wikipedia.org/wiki/Cron#CRON_expression
Should I just use code markup for the entire phrase? what would you suggest?

Reply:
replaced with `CRON expression` and no longer need to add this to the spellchecker